drivers/power_domain/power_domain_intel_adsp.c

```diff
@@ -63,20 +63,10 @@ static int pd_intel_adsp_pm_action(const struct device *dev, enum pm_device_acti
 	switch (action) {
 	case PM_DEVICE_ACTION_RESUME:
 		ret = pd_intel_adsp_set_power_enable(reg_bits, true);
-
-		if (ret == 0) {
-			pm_device_children_action_run(dev, PM_DEVICE_ACTION_TURN_ON, NULL);
-		}
-
```

Comment:
This can't be done until we have removed `PM_DEVICE_SYSTEM_MANAGED`. Otherwise if the target is built without device runtime children nodes will not get `TURN_ON` and `TURN_OFF` actions.

We will do this, but it has to be done after the `SYSTEM_MANAGED` is deprecated / removed.